x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/reducer.ts

```diff
@@ -72,7 +72,7 @@ export function getModelMemoryLimitErrors(mmlValidationResult: any): string[] |
     if (errorKey === 'min') {
       acc.push(
         i18n.translate('xpack.ml.dataframe.analytics.create.modelMemoryUnitsMinError', {
-          defaultMessage: 'Model memory limit cannot be lower than {mml}',
+          defaultMessage: 'Model memory limit is lower than estimated value {mml}',
```

Comment:
Is it worth hiding this message if the units are missing / invalid? So in this case, we'd only show the message about the units being invalid:

![image](https://user-images.githubusercontent.com/7405507/95109188-29729100-0734-11eb-9ad0-754fd6ac564e.png)

Reply:
I think it's worth keeping both for now so that the requirements are clear.